### ISSUE TITLE
[NFR]: Add precompiled binaries for Linux and macOS ARM64

### ISSUE THREAD

**Issue by Sn0wCrack** [phalcon/cphalcon#16553]
**Is your feature request related to a problem? Please describe.**
Not related to any existing issue.

**Describe the solution you'd like**
Provide ARM64 builds for Linux and macOS.

I believe GitHub has started offering support for ARM64 runners that could make this possible.

**Describe alternatives you've considered**
Currently cloning the repository and compiling as part of our initial environment setup, however this is a little slow, but not a huge issue as it only affects first time setup.

**Additional context**
Basically looking at switching over to development on the Apple Silicon, for our local environments we always download the the specific Phalcon version our production environment is using, however now we're cloning and compiling it locally instead.

Compiling seems to work exactly as expected and I haven't run into any particular issues with the binaries I've compiled myself.

**Comment by Sn0wCrack** [comment]
GitHub at some point between this issue opening and now has added support for macOS ARM64 runners:

https://github.com/actions/runner-images?tab=readme-ov-file#available-images

**Comment by niden** [comment]
Resolved in https://github.com/phalcon/cphalcon/pull/17001

Thank you @Sn0wCrack
